init.go

```
@@ -0,0 +1,16 @@
+package cli
+
+import "github.com/urfave/cli/v2"
+
+func NewInitCmd(config *Config) *cli.Command {
```

IMHO we don't need this command.

---

I think almost all CLIs have an init command and to me it's more intuitive that `stream config new`

---

AWS and Github do not 🙈 which are for me the most used, but anyway it's fine for me but I think:
- calling init should make the application created the default one
- if an application already exists, we should return an error (that's a suggestion)

---

i think just `stream config` would be ideal

---

works for me!